package.json

```diff
@@ -20,7 +20,7 @@
     "client-sessions": "^0.8.0",
     "cookie-parser": "^1.4.5",
     "cross-spawn": "^7.0.2",
-    "cypress": "5.3.0",
+    "cypress": "8.2.0",
```

Comment:
Big jump here. Any API changes that impact us?

Reply:
I think that I was very lucky with this one, I tested that Cypress ran and got the correct outputs seemed to work for me.